provider/resource_keycloak_oidc_identity_provider.go

```diff
@@ -62,6 +62,18 @@ func resourceKeycloakOidcIdentityProvider() *schema.Resource {
 			Required:    true,
 			Description: "Token URL.",
 		},
+		"issuer": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Issuer Identifier",
+			Default:     "",
```

Comment:
nitpick: This will always default to the zero-value for strings, which is an empty string.  So this line can be removed.

Reply:
thanks for the review! fixed as you suggested.